lib/widgets.php

```diff
@@ -126,3 +126,58 @@ function gutenberg_legacy_widget_settings( $settings ) {
 	return $settings;
 }
 add_filter( 'block_editor_settings', 'gutenberg_legacy_widget_settings' );
+
+
+function create_wp_area_post_type() {
+	register_post_type(
+		'wp_area',
+		array(
+			'labels'                => array(
+				'name'                     => _x( 'WordPress Area', 'post type general name' ),
+				'singular_name'            => _x( 'WordPress Area', 'post type singular name' ),
+				'menu_name'                => _x( 'WordPress Areas', 'admin menu' ),
+				'name_admin_bar'           => _x( 'WordPress Area', 'add new on admin bar' ),
+				'add_new'                  => _x( 'Add New', 'Block' ),
+				'add_new_item'             => __( 'Add New WordPress Area' ),
+				'new_item'                 => __( 'New WordPress Area' ),
+				'edit_item'                => __( 'Edit WordPress Area' ),
+				'view_item'                => __( 'View WordPress Area' ),
+				'all_items'                => __( 'All WordPress Areas' ),
+				'search_items'             => __( 'Search WordPress Areas' ),
+				'not_found'                => __( 'No WordPress area found.' ),
+				'not_found_in_trash'       => __( 'No WordPress areas found in Trash.' ),
+				'filter_items_list'        => __( 'Filter WordPress areas list' ),
+				'items_list_navigation'    => __( 'WordPress areas list navigation' ),
+				'items_list'               => __( 'WordPress areas list' ),
+				'item_published'           => __( 'WordPress area published.' ),
+				'item_published_privately' => __( 'WordPress area published privately.' ),
+				'item_reverted_to_draft'   => __( 'WordPress area reverted to draft.' ),
+				'item_scheduled'           => __( 'WordPress area scheduled.' ),
+				'item_updated'             => __( 'WordPress area updated.' ),
+			),
+			'public'                => false,
+			'show_ui'               => true,
```

Comment:
This should probably be false?

Reply:
Yes, show_ui should be false, it is true to make testing easier.